### PR TITLE
Add annotated examples for single/multiple fonts

### PR DESCRIPTION
**Pull request by RoelN**
Notably, a dropdown to change font with.

### CODE REVIEW

**Comment by RoelN** [comment]
Note that the order of fonts is nondeterministic, and can change with every build. @pascalw Any way to work around this? I'm thinking of maybe normalizing the 11ty object into something easier to use, like a flat array. WDYT?

**Comment by RoelN** [comment]
Also, not happy with duplicating the font selectors stuff. Maybe we should generate an extra _data file from the support, which contains the human readable font names, the CSS font name, and the CSS class, so a developer can pick the notation they need.

**Comment by pascalw** [comment]
As discussed offline:

* The non deterministic order seems to be caused by 11ty. 11ty adds global data files to the global data file in an non-deterministic order. Even if it didn't, relying on ordering of object keys is not very safe. So best is probably to generate a list of fonts somewhere, where we control the order of that list.
* Duplicating the selector stuff can be prevented by including the selectors for each font in `fonts.js`.

**Comment by RoelN** [comment]
I addressed the 2nd point and moved the name/classname generation to the support.

Regarding handling the nondeterministic stuff, I'll open an issue for this and address this in the near future!

**Comment by RoelN** [comment]
Issue to address the 1st point: https://github.com/kabisa/specimen-skeleton/issues/64

File: src/js/main.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const throttle = (fn, wait) => {
 // Set up FontFaceObserver
 let observers = [];
 for (const fontName of fontNames) {
```
[the remark this answers is not in the document]
Comment (pascalw) [reply src/js/main.js:40]: Oh also don't forget to update the version of the support library!

**Review by pascalw — APPROVED** [review]

File: src/js/main.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (RoelN) [reply src/js/main.js:40]: Will change names, and will update refernce after merge!

**Pull request merged**